src/main/java/net/minestom/server/event/inventory/InventoryPreClickEvent.java

```diff
@@ -17,23 +15,16 @@ public class InventoryPreClickEvent implements InventoryEvent, PlayerInstanceEve
 
```

Comment:
As discussed I think it would be a nice utility to have a `getClickedItem` which returns the item if the click is a single slot click or air otherwise. Often when making menus you only care about the single slot ones so I guess people would find usage.

Similarly its perhaps worth also adding a getSlot() which just proxies to Click#slot()?